internal/cli/deployment_list.go

```diff
@@ -281,6 +281,7 @@ func (c *DeploymentListCommand) Run(args []string) int {
 					details[0],
 					startTime,
 					completeTime,
+					b.Preload.DeployUrl,
```

Comment:
I think given this bug: https://github.com/hashicorp/waypoint/issues/1608
We should only be showing the deploy URL with the `verbse` flag. Otherwise I think deployment list might be pretty difficult to format and look nice until that is fixed.